openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -11,7 +11,7 @@
   // rules to overcome any poorly defined, scoped, or architected edX LMS-based CSS. Ideally this file will be removed when the LMS Front End is re-visited.
 
   // headers
-  .self-assessment__display__header, .peer-assessment__display__header, .step__header {
+  .self-assessment__display__header, .peer-assessment__display__header, .step__header, .staff-assessment__display__header {
```

Comment:
@clrux They are already in the ORA repo. In my opinion, refactoring the ORA CSS files to get rid of _shame.scss is beyond the scope of this change.